src/start.sh

```diff
@@ -27,20 +26,24 @@ echo "  [i] Starting docker specific checks & setup for docker pihole/pihole"
 
 # If PIHOLE_UID is set, modify the pihole user's id to match
 if [ -n "${PIHOLE_UID}" ]; then
-    currentId=$(id -u ${username})
```

Comment:
a leftover from when it was a function, ${username} should be replaced by pihole